### PR TITLE
port publish mentioned on readme

### PR DESCRIPTION
**Pull request by matzew**
adding -p option to access the server

### CODE REVIEW

File: server-mysql/README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ First start a MySQL instance using the MySQL docker image:
 
 Start a Keycloak instance and connect to the MySQL instance:
 
-    docker run --name keycloak --link mysql:mysql jboss/keycloak-mysql
+    docker run -it -p 8080:8080 --name keycloak --link mysql:mysql jboss/keycloak-mysql
```
Comment (goldmann) [inline server-mysql/README.md]: I would better not commit this, but maybe @mstruk or @abstractj can say what they prefer?

Comment (goldmann) [inline server-mysql/README.md]: Or @stianst, of course :)

**Comment by stianst** [comment]
We should make it consistent for all images, but it makes more sense to me to drop '-p 8080:8080' from the main Keycloak image than to add it here. The readme for the WildFly image doesn't have it and we should be consistent. I reckon that's what makes sense as binding to localhost:8080 prevents running multiple WF/KC instances.

**Comment by stianst** [comment]
Decided to remove -it -p 8080:8080 from main image as well, so closing this

**Pull request closed without merging**